handbook/company/pricing-features-table.yml

```diff
@@ -710,7 +695,16 @@
     - description: Progressively enhance from Nudge to DDM-based OS updates.
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/17295
     - description: Automatically update Windows after the end user reaches a deadline.
-- industryName: Cross-platform remote lock and wipe
+- industryName: Native macOS update reminders
```

Comment:
tbh we wanted to consolidate this with "Enforce OS updates" as it's part of the same feature, but we didn't want to lose a checkmark in Fleet Free, so we decided to leave it.

Reply:
@mike-j-thomas I think this one is worth removing/consolidating. What looks bad if we remove a Fleet Free checkbox?

This row is about sending a macOS host this low-level MDM command: https://developer.apple.com/documentation/devicemanagement/scheduleosupdatecommand

I think IT admins will this row w/ the new Software Update feature that's built on Apple's new DDM protocol (#17295). This is a premium feature.

Reply:
Thanks, @noahtalerman. I consolidated this with "Enforce OS updates" in this commit https://github.com/fleetdm/fleet/pull/20609/commits/a44d0263afde9e69212aca71caf56b2cf4fa4e85